substrate/frame/balances/src/tests/mod.rs

```diff
@@ -132,9 +138,10 @@ impl Config for Test {
 	type ReserveIdentifier = TestId;
 	type WeightInfo = ();
 	type RuntimeHoldReason = TestId;
+	type RuntimeFreezeReason = RuntimeFreezeReason;
```

Comment:
Shouldn't this be `TestId`?

Reply:
probably me or oliver did too much copy pasta :D I will double check

Reply:
actually probably not, it should be the default one.